renovate.json

```diff
@@ -1,13 +1,15 @@
 {
+  "automerge": false,
+  "branchPrefix": "renovate/",
   "extends": [
     "config:base"
   ],
-  "prConcurrentLimit": 2,
-  "schedule": "every weekend",
-  "automerge": false,
-  "branchPrefix": "renovate/",
   "includePaths": [
     "packages/fxa-payments-server/package.json",
-    "packages/fxa-payments-server/Dockerfile"
-  ]
+    "packages/fxa-payments-server/Dockerfile",
+    "packages/fxa-support-server/package.json"
```

Comment:
Unless I'm very behind and/or out of the loop, there's no `fxa-support-server`.  Did you mean `fxa-support-panel`?

Reply:
doh, yes, search/replace fail